Resources/Views/Admin/presentations.leaf

```diff
@@ -0,0 +1,16 @@
+<h1>Speakers</h1>
```

Comment:
I was using _ to denote snippets in the home page example, not sure if you want to do the same here.  i.e. _presentations.leaf.  Also, should it be <h1>Presentations</h1>?